index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingrese su nombre:');
+alert(`Hola, ${nombreUsuario}!`);
```

Comment:
Sería mejor no hacer interpolación de strings 
```suggestion
alert(`Hola, ` + nombreUsuario);
```